e2e/e2e-script.sh

```diff
@@ -247,5 +242,13 @@ else
     exit 1
 fi
 
+waitForDeleteStartTime=$(date +%s)
+
+az vmss delete -g $MC_RESOURCE_GROUP_NAME -n $VMSS_NAME
```

Comment:
the delete sequence takes ~1minute to run, doing some more test runs to see if cloud controller manager deletes the node after certain amount of time, if yes, will add --no-wait to vmss delete and remove the kubectl delete.